.github/workflows/release.yml

```diff
@@ -17,4 +17,38 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+      - name: Cache Maven packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+          server-id: ossrh-distro
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_PASSWORD
+          gpg-private-key: ${{ secrets.GPG_KEY }}
+          gpg-passphrase: MAVEN_GPG_PASSPHRASE
+      - name: Configure Git user
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+      - name: Release and deploy with Maven
+        run: mvn -B --file pom.xml release:prepare release:perform -Prelease -DdevelopmentVersion=${{ github.event.inputs.dev-version }} -DreleaseVersion=${{ github.event.inputs.release-version }}
```

Comment:
How are defined `github.event.inputs.dev-version` and `github.event.inputs.release-version`?

Reply:
They are defined here: https://github.com/symphonyoss/messageml-utils/pull/244/files/799cf104ba587c7b0d0bec49bae351ec8e8d7d40#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R5
as manual inputs for the job's execution